DEPRECATIONS.md

```diff
@@ -0,0 +1,27 @@
+<!--lint disable no-file-name-mixed-case -- mixed case in the file name for version -->
+
+# Deprecations
+
+Introducing lists containing deprecations and feature flags to be removed in the nex major version.
+
+💁 Please follow these steps to safely upgrade your Design System to Spirit Design System to the next major version.
```

Comment:
I would remove this sentence; there are no steps in this file.

Reply:
Removed.